spec/node/asciidoctor.spec.js

```diff
@@ -468,6 +471,38 @@ Content 2`;
       }
     });
 
+    it('should be able to unregister multiple statically-registered extension groups as Array', () => {
+      var extensions = asciidoctor.Extensions;
+      try {
+        extensions.register('foo', function () {
+          this.blockMacro(function () {
+            this.named('foo');
+            this.process((parent) => {
+              return this.createBlock(parent, 'paragraph', 'foo means foo');
+            });
+          });
+        });
+        extensions.register('bar', function () {
+          this.blockMacro(function () {
+            this.named('bar');
+            this.process((parent) => {
+              return this.createBlock(parent, 'paragraph', 'bar or bust');
+            });
+          });
+        });
+        let groups = extensions.getGroups();
+        expect(groups).toBeDefined();
+        expect(Object.keys(groups).length).toBe(2);
+        expect(Object.keys(groups)).toEqual(['foo', 'bar']);
+        extensions.unregister(['foo', 'bar']);
+        groups = extensions.getGroups();
+        expect(groups).toBeDefined();
+        expect(Object.keys(groups).length).toBe(0);
```

Comment:
Tests are failing, I didn't debug but `Object.keys(groups).length` equals 1 not 0: https://travis-ci.org/asciidoctor/asciidoctor.js/jobs/358264467